sdk/apimanagement/api-management-custom-widgets-scaffolder/bin/execute.cjs

```diff
@@ -523,8 +523,8 @@ let mixin;
 function cliui(opts, _mixin) {
     mixin = _mixin;
     return new UI({
-        width: (opts === null || opts === void 0 ? void 0 : opts.width) || getWindowWidth(),
-        wrap: opts === null || opts === void 0 ? void 0 : opts.wrap
+        width: (opts === null || opts === undefined ? undefined : opts.width) || getWindowWidth(),
```

Comment:
not sure if this is expected?

Reply:
I didn't change anything but run the build, so I assume their logic changed at some point.